views/pdf/PaymentInvoice.pug

```diff
@@ -320,7 +320,7 @@ html
       .col.col-24
```

Comment:
The Receipt should show only invoice number , date , invoice total (small font)
Total paid (Big Font)
There is no items list